site/src/sphinx/advanced-unit-testing.rst

```diff
@@ -0,0 +1,128 @@
+.. _advanced-unit-testing:
+
+Unit-testing ``Client`` and ``Service``
+=======================================
+
+A unit test of a :api:`Client` or a :api:`Service` will require you to prepare two objects:
+
+- :api:`ClientRequestContext` or :api:`ServiceRequestContext`
+- :api:`HttpRequest` or :api:`RpcRequest`
+
+:api:`ClientRequestContext` or :api:`ServiceRequestContext` is a more complex object with many properties than
+:api:`HttpRequest` or :api:`RpcRequest`, and thus Armeria provides the API dedicated to building a fake
```

Comment:
to build?

Reply:
Fixed

Reply:
Realized that 'building' is correct here - 'dedicated to something'

Reply:
Oops. Sorry about that. :-)